README.md

```diff
@@ -1,56 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Basic Webpack Setup
 
+> This is a basic webpack setup project demo.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html
+- Webpack
+- CSS
+- Git and Github
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link]()
 
 ## Getting Started
 
```

Comment:
- [ ] For the `Getting Started` section, Please add the instruction on how to clone your repo, install dependencies libraries `npm install`, run your build `npm run build`, and run your dev server `npm run start`. To make your readme file professional and user friendly.